Docs/source/usage/parameters.rst

```diff
@@ -1452,6 +1452,17 @@ Grid initialization
     the field solver. In particular, do not use any other boundary condition
     than periodic.
 
+* ``warpx.maxlevel_extEMfield_init`` (default is maximum number of levels in the simulation)
+    With this parameter, the externally applied electric and magnetic fields with parameters
+    ``warpx.E_ext_grid_init_style="constant"``,
```

Comment:
This might be too much detail in this documentation. I think it might be cleaner to say something like
"With this parameter, any externally applied electric and magnetic fields will not be used to initialize the fields for levels greater than ``warpx.maxlevel_extEMfield_init``. ..."
My concern is that if in the future a new method of initializing the field is implemented, this comment will become out of date and incomplete.